examples/segnet/README.md

```diff
@@ -14,7 +14,7 @@ python calc_weight.py
 First, move to this directory (i.e., `examples/segnet`) and run:
 
 ```
-CHAINER_SEED=2017 CHAINER_TYPE_CHECK=0 python -OO -W ignore train.py --gpu 0
+python train.py --gpu 0
```

Comment:
Can the training code work with CPU? I think it is better to make `--gpu`  optional. How about `$ python train.py [--gpu <gpu>]`?